packages/core/src/Modules/Cashier/Components/Form/payment-agent-withdraw-form.jsx

```diff
@@ -130,162 +130,169 @@ const RadioInput = ({ touched, errors, field, values, ...props }) => (
     </Radio>
 );
 
-class PaymentAgentWithdrawForm extends React.Component {
-    componentDidMount() {
-        this.props.onMount();
-    }
+const PaymentAgentWithdrawForm = ({
+    amount,
+    balance,
+    currency,
+    error,
+    is_loading,
+    is_try_withdraw_successful,
+    is_withdraw_successful,
+    onMount,
+    payment_agent_id,
+    payment_agent_list,
+    payment_agent_name,
+    requestTryPaymentAgentWithdraw,
+    resetPaymentAgent,
+    verification_code,
+}) => {
+    React.useEffect(() => {
+        onMount();
 
-    componentWillUnmount() {
-        this.props.resetPaymentAgent();
-    }
+        return () => {
+            resetPaymentAgent();
+        };
+    }, []);
```

Comment:
No `eslint` warnings for exhaustive deps?

Reply:
There is in fact, I will add the eslint disable in.

Reply:
How about adding them? Does it break anything?